src/Tribe/Aggregator.php

```diff
@@ -58,8 +58,7 @@ class Tribe__Events__Aggregator {
 	 */
 	public static function instance() {
 		if ( ! isset( self::$instance ) ) {
```

Comment:
So `isset()` returns `false` when we are dealing with a variable with `value === null`, so it's a safer way to check for `is_null`, prevents notices.